src/main.rs

```diff
@@ -101,6 +103,12 @@ enum Matchers {
     Myers,
 }
 
+#[derive(Debug, Eq, PartialEq, RustcDecodable)]
+enum Output {
+    NoOutput, // Default.
+    Json,
```

Comment:
Good question, I can never figure out the Rust conventions, but I think this *can* be all caps, in which case the other enums in main should be too.